openshift_scalability/utils.py

```diff
@@ -34,12 +34,29 @@ def oc_command(args, globalvars):
 def login(user,passwd,master):
     return subprocess.check_output("oc login --insecure-skip-tls-verify=true -u " + user + " -p " + passwd + " " + master,shell=True)
 
+def get_route():
+    default_proj = subprocess.check_output("oc project default", shell=True)
+    localhost = subprocess.check_output("ip addr show eth0 | grep \"inet \" | awk '{print $2}' | cut -d/ -f1", shell=True).rstrip()
+    router_name = subprocess.check_output("oc get pod | awk '/router/ {print $1;}'", shell=True).rstrip()
+    router_ip = subprocess.check_output("oc describe pod %s | awk '/IP/ {print $2}'" % router_name, shell=True).rstrip()
+    routes_output = subprocess.check_output("oc get route --all-namespaces --no-headers | awk '/example/ {print $3;}'", shell=True)
+    routes_list = [y for y in (x.strip() for x in routes_output.splitlines()) if y]
+    return localhost, router_ip, routes_list
+
 
 def create_template(templatefile, num, parameters, globalvars):
     if globalvars["debugoption"]:
         print "create_template function called"
 
-    namespace = globalvars["namespace"]
+    if globalvars["autogen"]:
+        localhost, router_ip, jmeter_ips = get_route()
+	extra_param = {}
+	extra_param['GUN'] = localhost
+	parameters.append(extra_param.copy())
+        size = 3 # How many webservices to test per instance
```

Comment:
Changed 1de8c27b0a2a9fb914b414e499d1c1cacc6f7e42, do you know why the key index doesn't work like a normal dict inside a list for parameters?